### ISSUE TITLE
Double braces fail with braces in text

### ISSUE THREAD

**Issue by jonas-eschle** [FlamingTempura/bibtex-tidy#410]
Currently, the " can be replaced by {, but this will render titles with special characters problematic.
Is there an option available, or can one be added, to have a " be replaced by {" for example?

**Comment by FlamingTempura** [comment]
Could you give an example of some bibtex where this would be useful and what you'd expect the output to be?

**Comment by jonas-eschle** [comment]
Hi @FlamingTempura , just wondering if you could reproduce it and maybe had the time to take a look at it?

**Comment by FlamingTempura** [comment]
Hey - thanks for reporting back with reproduction steps, I've not had a chance to look into this yet but can confirm it's a bug in bibtex-tidy.

Can't promise when I'll be able to fix it. I'd welcome a pull request if it's something you'd be up for investigating.

**Comment by FlamingTempura** [comment]
This should now be fixed in web UI and v1.12.0.

Output for the above example:
```bibtex
@article{LHCb-PAPER-2020-001,
	title        = {{Search for the rare decays \mbox{\decay{\Bs}{\epem}} and \mbox{\decay{\Bd}{\epem}}}},
	author       = {Aaij, R. and others},
	year         = 2020,
	journal      = {Phys. Rev. Lett.},
	volume       = 124,
	pages        = 211802,
	doi          = {10.1103/PhysRevLett.124.211802},
	collaboration = {LHCb collaboration},
	report       = {{LHCb-PAPER-2020-001 CERN-EP-2020-023}},
	eprint       = {2003.03999},
	archiveprefix = {arXiv},
	primaryclass = {hep-ex}
}
```

If this hasn't solved the issue then please reopen the ticket.
